doc/api/index.rst

```diff
@@ -43,3 +51,10 @@ Ceph Object Store APIs
 .. _S3-compatible API: ../radosgw/s3/
 .. _Swift-compatible API: ../radosgw/swift/
 .. _Admin Ops API: ../radosgw/adminops
+
+Ceph Command API
+================
+
+See `mon_command_api`_.
+
+.. _mon_command_api: ./mon_command_api
```

Comment:
Please do not reference documents by path, use the `:ref:` directive. This has been very problematic in the past and has caused broken links when docs move. It is explained in the contributing guidelines: https://github.com/ceph/ceph/blob/master/doc/start/documenting-ceph.rst#toc-and-hyperlinks